app/admin/applications/new/page.tsx

```diff
@@ -5,6 +5,7 @@ import { useCallback, useState } from "react";
 import { AppTypes, AppType } from "@/db/constants";
 import TwitterForm from "@/components/TwitterForm";
 import CapitalOneForm from "@/components/CapitalOneForm";
+import PlaidForm from "@/components/PlaidForm";
```

Comment:
Revert all changes that isn't in the vellum directory. We don't want Plaid to be user facing, the end goal is Capital One still